mkosi

```diff
@@ -107,6 +107,7 @@ def create_image(args, workspace):
     print_step("Creating partition table...")
 
     f = tempfile.NamedTemporaryFile(dir = os.path.dirname(args.output), prefix='.mkosi-')
+    subprocess.run(["chattr", "+C", f.name], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=True)
     f.truncate(image_size(args))
```

Comment:
I wish I could find a function to change file attributes in the python standard library.
Unfortunately, without that, invoking `chattr` was the simplest way.